### PR TITLE
Further changes for #28

### PR DESCRIPTION
**Pull request by mayankchhabra**
Yo @nolim1t, did some changes for #28.

- Added `getumbrel/tor:v0.4.1.9` image for Tor instead of building on device (uses your Dockerfile for the Tor container https://github.com/getumbrel/docker-tor)
- Removed `build/volume-permissions` as it wasn't used anywhere
- Pull docker images inside `configure-box.sh`
- Use containerized Tor to generate hashed password in `configure-box.sh` as there is no Tor installed on the system anymore
- Removed pre-defined Tor password from `torrc`
- Commented out TLS options in `lnd.conf` as they errored and lnd would just not start with them. Not really sure about the options, but do they work for you?

### CODE REVIEW

File: lnd/lnd.conf [hunk 1]
```diff
@@ -16,10 +16,10 @@ accept-keysend=true
 [Routing]
 routing.assumechanvalid=1
 ; Extra TLS
-tlsextradomain=lnd
-tlsextraip=10.11.1.2
+; tlsextradomain=lnd
+; tlsextraip=10.11.1.2
 ; Unsure if this is a 0.10.0 command (make mental note to test this)
-tlsautorefresh=1
+; tlsautorefresh=1
```
Comment (nolim1t) [inline lnd/lnd.conf]: Why comment this out? its a useful feature to have

File: lnd/lnd.conf [hunk 2]
```diff
@@ -16,10 +16,10 @@ accept-keysend=true
 [Routing]
 routing.assumechanvalid=1
 ; Extra TLS
-tlsextradomain=lnd
-tlsextraip=10.11.1.2
+; tlsextradomain=lnd
+; tlsextraip=10.11.1.2
```
Comment (nolim1t) [inline lnd/lnd.conf]: Also regarding commenting this out too

File: tor/torrc [hunk 3]
```diff
@@ -1,6 +1,3 @@
 # Bind only to "10.11.5.1" which is the tor IP within the container
 SocksPort   10.11.5.1:29050
 ControlPort 10.11.5.1:29051
-
-# Tor Passwords
-HashedControlPassword 16:50A873DF18C00F4A6048BF1CEF7E7AA66478F0B5134DA4369D80657F26
```
Comment (nolim1t) [inline tor/torrc:6]: Please keep this in for now. I'll remove it later

**Comment by nolim1t** [comment]
the tls options work for me

**Comment by mayankchhabra** [comment]
Strange, it doesn't work for me at all on all 3 chains. Can you try on a fresh install?

**Comment by mayankchhabra** [comment]
Can we merge this today? Want #28 to be merged before I open a PR for OTA updates. As for the tls options, we can uncomment them later.

**Comment by nolim1t** [comment]
Can we leave the torrc config alone.. the extra line isn't harming anything

**Comment by nolim1t** [comment]
tlsautorefresh is enabled for me and works

**Comment by mayankchhabra** [comment]
> Can we leave the torrc config alone.. the extra line isn't harming anything

Isn't hardcoding a Tor password a security risk? Plus is it even being used anywhere since a new password is generated on the device anyway? 

> tlsautorefresh is enabled for me and works

Ok then I'll uncomment it

File: lnd/lnd.conf (hunk 2 above)
Comment (nolim1t) [reply lnd/lnd.conf]: Can we keep this comment as it is?

File: tor/torrc (hunk 3 above)
Comment (nolim1t) [reply tor/torrc:6]: Also this too. THanks.

Rest looks good otherwise

**Review by nolim1t — CHANGES_REQUESTED** [review]
LGTM.. just a few things I'd like to keep off for now

**Review by nolim1t — APPROVED** [review]

**Comment by nolim1t** [comment]
Merged into draft branch. Thanks for the tor container too

**Pull request merged**